source/features/ci-link.tsx

```diff
@@ -27,7 +27,11 @@ async function init(): Promise<false | void> {
 	}
 
 	// Append to title (aware of forks and private repos)
-	select('[itemprop="name"]')!.after(icon);
+	const privateLabel = select('[itemprop="name"] + .Label');
+	(privateLabel ?? select('[itemprop="name"]'))!.after(icon);
+	if (privateLabel) {
+		icon.classList.add('ml-2');
+	}
```

Comment:
Can you move the spacing to the CSS? It would be easier to specify it there and it also simplifies this code to:

```suggestion
	select('[itemprop="name"]')!.parentElement!.append(icon);
```